docking.js

```diff
@@ -2018,8 +2018,14 @@ var DockManager = class DashToDock_DockManager {
             }
 
             const mainDockProperties = {};
-            if (dock === this.mainDock)
-                mainDockProperties.onComplete = callback;
+            if (dock === this.mainDock && callback !== undefined) {
```

Comment:
```suggestion
            if (dock === this.mainDock && callback) {
```
Otherwise we'd could call a `null` callback too.

Reply:
Sure, not important. Although I think a cleaner solution is to modify the caller to pass in `() => {}` since it's an expected parameter.

Reply:
I'm not sure null as a callback is something we need to set a precedent for or support. Anyway, I've solved it in https://github.com/micheleg/dash-to-dock/pull/1843 instead.